server/odc-service/src/main/java/com/oceanbase/odc/service/connection/logicaldatabase/BadExpressionException.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.connection.logicaldatabase;
+
+import org.springframework.http.HttpStatus;
+
+import com.oceanbase.odc.core.shared.constant.ErrorCode;
+import com.oceanbase.odc.core.shared.exception.HttpException;
+
+/**
+ * @Author: Lebie
+ * @Date: 2024/4/23 14:03
+ * @Description: []
+ */
+public class BadExpressionException extends HttpException {
```

Comment:
shall we move this exception to `com.oceanbase.odc.core.shared.exception`?

Reply:
I prefer to keep it in the service layer as it is a specific exception but not a general/common exception like `BadRepquest`, etc.